R/bambu.R

```diff
@@ -115,28 +115,27 @@
 #' se <- bambu(reads = test.bam, annotations = gr, 
 #'     genome = fa.file,  discovery = TRUE, quant = TRUE)
 #' @export
-bambu <- function(reads = NULL, rcFile = NULL, rcOutDir = NULL,
+bambu <- function(reads, rcOutDir = NULL,
     annotations = NULL, genome = NULL, stranded = FALSE, ncore = 1, NDR = 0.1,
     yieldSize = NULL, opt.discovery = NULL, opt.em = NULL, trackReads = FALSE, 
     returnDistTable = FALSE, discovery = TRUE, quant = TRUE, fusionMode = FALSE, 
     verbose = FALSE, lowMemory = FALSE) {
```

Comment:
```suggestion
bambu <- function(reads,
    annotations = NULL, genome = NULL, NDR = 0.1, stranded = FALSE, rcOutDir = NULL,
     opt.discovery = NULL, opt.em = NULL, fusionMode = FALSE, discovery = TRUE, quant = TRUE, trackReads = FALSE,     returnDistTable = FALSE,  ncore = 1,  yieldSize = NULL, 
     lowMemory = FALSE, verbose = FALSE) {
```

Reply:
resolved in 85346d939f9089fe04797147d066dfd063c320f6

Reply:
this change will make this a major release update